fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedIndexMeta.java

```diff
@@ -89,18 +90,19 @@ public class MaterializedIndexMeta implements Writable, GsonPostProcessable {
     private boolean isColocateMVIndex = false;
 
     private Expr whereClause;
-    private Set<Long> updateSchemaBackendId = new HashSet<>();
+    private Set<Long> updateSchemaBackendId;
```

Comment:
`private Set<Long> updateSchemaBackendId = new HashSet<>();`

will this not work for gson deserialize?

Reply:
Yes

Reply:
how about create the Set in postGson? on-demand creating always introduces one level of indirection.

Reply:
This field has a quite low access frequency，so I also want to reduce the memory usage.